params.go

```diff
@@ -43,6 +43,7 @@ type Params struct {
 	Secrets                         SecretsParams       `json:"secrets,omitempty" yaml:"secrets,omitempty"`
 	Configs                         ConfigsParams       `json:"configs,omitempty" yaml:"configs,omitempty"`
 	VolumeMounts                    []VolumeMountParams `json:"volumemounts,omitempty" yaml:"volumemounts,omitempty"`
+	CertificateSecret               string              `json:"certificatesecret,omitempty" yaml:"certificatesecret,omitempty"`
```

Comment:
Is this in order to use the shared wildcard? It would be nice if we can update the default behaviour when that wildcard is omni-present so we don't need to update manifests in tons of applications.

Reply:
Makes sense, I'll do it the other way around then.

Reply:
Actually I was planning on setting this on the defaults in estafette-ci-api.
So we can specify the name of this secret per each cluster.

Reply:
Ah that makes sense!